### PR TITLE
HARP-2497: Removed ZoomLevel from decoding.

### PR DESCRIPTION
**Pull request by amsgames94**

### CODE REVIEW

File: @here/harp-omv-datasource/lib/IGeometryProcessor.ts [hunk 1]
```diff
@@ -98,14 +94,12 @@ export interface IGeometryProcessor {
      * @param layerName The name of the enclosing layer.
      * @param geometry The list of polygons.
      * @param env The enviroment containing the properties of the geometry.
-     * @param storageLevel The storage level of the data.
-     * @param displayZoomLevel The desired display zoom level.
+     * @param storageLevel The storage level of the data
```
Comment (StefanDachwitz) [inline @here/harp-omv-datasource/lib/IGeometryProcessor.ts]: missing colon

File: @here/harp-mapview/lib/MapView.ts [hunk 2]
```diff
@@ -1187,6 +1187,17 @@ export class MapView extends THREE.EventDispatcher {
         return this.m_zoomLevel;
     }
 
+    /**
+     * Returns the storage level for the given camera setup.
```
Comment (StefanDachwitz) [inline @here/harp-mapview/lib/MapView.ts]: Please add something like this: "Actual storage level of the rendered tile data also depends on [[Tile.storageLevelOffset]]."

File: @here/harp-omv-datasource/lib/IGeometryProcessor.ts (hunk 1 above)
Comment (amsgames94) [reply @here/harp-omv-datasource/lib/IGeometryProcessor.ts]: Good catch. Will fix.

File: @here/harp-mapview/lib/MapView.ts (hunk 2 above)
Comment (amsgames94) [reply @here/harp-mapview/lib/MapView.ts]: Will do.

**Comment by amsgames94** [comment]
@heremaps-bot retest this please

**Pull request merged**